Tutorials/intro-to-simulation/intro-to-simulation.py

```diff
@@ -0,0 +1,125 @@
+"""
+intro_to_simulation.py: Demonstrate usage of the simulation feature
+Author: Tomer Feld - Quantum Machines
+Created: 26/06/2022
+
+"""
+import numpy as np
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm import SimulationConfig
+from qm import LoopbackInterface
+from qm.qua import *
+import matplotlib.pyplot as plt
+from configuration import *
+
+qmm = QuantumMachinesManager()
```

Comment:
Maybe add IP?

Reply:
Which IP?

Reply:
A variable `QOP_IP` which will be set to `None` or `127.0.0.1` for now?